src/internt-vedtak/typer/vilkårperiode.ts

```diff
@@ -110,4 +111,5 @@ export const typeStønadsperiodeTilTekst: Record<TypeStønadsperiode, string> =
     UTDANNING: 'Utdanning',
     REELL_ARBEIDSSØKER: 'Reell arbeidssøker',
     INGEN_AKTIVITET: 'Ingen aktivitet',
+    INGEN_UTDANNING: 'Ingen utdanning',
```

Comment:
Det er et poeng at vi viser nøyaktig samme teksten som det saksbehandlerne faktisk vurderer, slik at det blir null rom for mistolkning.